src/content/docs/codestream/how-use-codestream/request-feedback.mdx

```diff
@@ -3,206 +3,211 @@ title: Request feedback on CodeStream
 metaDescription: "How to review and request feedback from your teammates, as well as how to make direct code changes."
 ---
 
-CodeStream's Feedback Requests are powerful enough to use for traditional
+New Relic CodeStream's feedback requests are powerful enough to use for traditional
 end-of-cycle code reviews, but at the same time they're so easy and flexible
 that you can use them throughout the development process to get quick feedback
-on your work in progress. Even if you haven't committed your changes!
+on your work in progress. You can even use feedback requests for uncommitted your changes.
 
 Traditional code review happens at the end of the development cycle, when you’re
 looking to get the changes merged. Not only are end-of-cycle code reviews much
 more burdensome on your teammates, but you run the risk of identifying issues so
 late in the game that you end up having to decide between blowing up your
 schedule or taking on technical debt.
 
-Whether you’re at the very beginning of a project, with just some stubbed out
+Whether you’re at the beginning of a project, with just some stubbed out
```

Comment:
No idea what `stubbed` means in this context. I could just be ignorant, but I think there's a better word to be found here.

Reply:
Fixing this is beyond the scope of this copyedit pass